### ISSUE TITLE
See more than one day when filtering by user

### ISSUE THREAD

**Issue by cailie** [pulseenergy/panorama#1]
I want to see further back -- ideally as far back as I like. But a week would be practical.

Not that I'm spying on anyone. I would just like to understand the evolution of related changes across repos.

**Comment by wachunga** [comment]
I agree. Unfortunately, we're a bit constrained by the GitHub API but it may be possible without hitting our rate limit.
